.gitmodules

```diff
@@ -1,6 +1,6 @@
 [submodule "clio-test-data"]
 	path = clio-test-data
-	url = git@github.com:chamons/clio-test-data.git
```

Comment:
Seems unnecessary diff. Revert?

Reply:
this is needed to build on VSTS CI. the ssh link doesn't work.